### PR TITLE
Pahender/sre 1993

### PR DESCRIPTION
**Pull request by phender**
### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate watchers.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1063,7 +1063,7 @@ pipeline {
                         }
                     }
                 } // stage('Fault inection testing on EL 8.8')
-                stage('Test RPMs on EL 8.6') {
+                stage('Test RPMs on EL 8.8') {
```
Comment (brianjmurrell) [inline Jenkinsfile:1066]: The RPMs are already being tested on EL 8.8 in the Functional test stages.  The entire point of this stage is to test the RPMs compatibility with EL 8.6.

File: Jenkinsfile [hunk 2]
```diff
@@ -1083,7 +1084,7 @@ pipeline {
                         }
                     }
                 } // stage('Test CentOS 7 RPMs')
-                stage('Test RPMs on Leap 15.4') {
+                stage('Test RPMs on Leap 15.5') {
```
Comment (brianjmurrell) [inline Jenkinsfile:1087]: Ditto for this stage.  This one is more informational than the EL 8.6 stage as support for 15.4 is ending soon but it's still useful for us to know and to be able to tell customers who may not be able to switch as quickly as SUSE would like them to that our basic sanity testing still passes on 15.4.

File: Jenkinsfile [hunk 3]
```diff
@@ -1097,8 +1098,8 @@ pipeline {
                          * additionally for this use-case, can't override
                            ftest_arg with this :-(
                         script {
-                            'Test RPMs on Leap 15.4': getFunctionalTestStage(
-                                name: 'Test RPMs on Leap 15.4',
+                            'Test RPMs on Leap 15.5': getFunctionalTestStage(
+                                name: 'Test RPMs on Leap 15.5',
```
Comment (brianjmurrell) [inline Jenkinsfile:1102]:
```suggestion
                            'Test RPMs on Leap 15.4': getFunctionalTestStage(
                                name: 'Test RPMs on Leap 15.4',
```

**Review by brianjmurrell — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (phender) [reply Jenkinsfile:1066]: There are no Build RPM stage for EL 8.6 or Leap 15.4 (e.g. https://build.hpdd.intel.com/job/daos-stack/job/daos/job/release%252F2.4/455/pipeline-graph/) and there are no 15.4 RPMs in the artifacts: https://build.hpdd.intel.com/job/daos-stack/job/daos/job/release%252F2.4/455/artifact/artifacts/leap15/

**Pull request closed without merging**